src/ctap2/mod.rs

```diff
@@ -1298,12 +1432,20 @@ pub(crate) fn configure_authenticator(
         let resp = dev.send_cbor_cancellable(&authcfg, alive);
         match resp {
             Ok(()) => {
-                callback.call(Ok(ManageResult::Success));
-                break;
+                let auth_info = unwrap_option!(dev.refresh_authenticator_info(), callback);
+                send_status(
+                    &status,
+                    StatusUpdate::InteractiveManagement(InteractiveUpdate::AuthConfigUpdate((
+                        AuthConfigResult::Success(auth_info.clone()),
+                        Some(pin_uv_auth_result),
+                    ))),
+                );
+                return true;
             }
             Err(e) => {
                 handle_errors!(e, status, callback, pin_uv_auth_result, skip_uv);
             }
         }
     }
+    false
```

Comment:
And here?

Also, we don't seem to check these return codes. IIRC they were needed to work around some deficiency in the USB device selector (maybe a race condition?) where we needed to send a late `SelectedToken` message.